composer.json

```diff
@@ -14,13 +14,13 @@
         "monolog/monolog": "^3.0",
         "php": ">=8.1.0",
         "psr/log": "^2.0 || ^3.0",
-        "symfony/console": "^6.0",
-        "symfony/dependency-injection": "^6.0",
-        "symfony/finder": "^6.0",
-        "symfony/process": "^6.0",
-        "symfony/yaml": "^6.0",
-        "symfony/http-client": "^6.0",
-        "symfony/http-foundation": "^6.0",
+        "symfony/console": "^6.0 || ^7.0",
+        "symfony/dependency-injection": "^6.0 || ^7.0",
+        "symfony/finder": "^6.0 || ^7.0",
+        "symfony/process": "^6.0 || ^7.0",
+        "symfony/yaml": "^6.0 || ^7.0",
+        "symfony/http-client": "^6.0 || ^7.0",
+        "symfony/http-foundation": "^6.0 || ^7.0",
         "consolidation/self-update": "~2.1.0"
```

Comment:
`consolidation/self-update` does not support Symfony 7. The upcoming [3.0.0](https://github.com/consolidation/self-update/releases/tag/3.0.0-alpha2) does.

Reply:
Thanks @saitho 
I updated the PR.